contents/handbook/company/diversity.md

```diff
@@ -1,44 +1,64 @@
+
+
+## How diversity helps us
 
-There are tens of millions of people working in software development globally, and greater diversity means a greater understanding of our users.
+At PostHog, we view diversity as a tactic, like paying people towards the top of the market, or communicating company goals to set context for our team. There is plenty of research into the link between highly diverse teams and increased [performance](https://www.ucdenver.edu/docs/librariesprovider68/default-document-library/jmna-articles-bonuscontent-2.pdf?Status=Temp&sfvrsn=84c0fb9_2) and [innovation](https://www.bcg.com/en-us/publications/2018/how-diverse-leadership-teams-boost-innovation). 
 
-## Diversity as a tactic
+In order to build the most diverse team, we have introduced the [Rooney rule](https://en.wikipedia.org/wiki/Rooney_Rule) to our Recruitment process. Originally implemented by the National Football League (NFL) but increasingly used by companies, the Rooney rule requires at least one person of an underrepresented minority to be considered for every open position.
 
-We don't explicitly set a target for diversity as a company objective or value. Instead, we view diversity as a tactic like paying people towards the top of market, or communicating company goals to set context for our team. Like both of those things, diversity will lead to great performance.
+In the context of tech and startups, categories of people who are underrepresented include those who identify as: 
+* A person of color
+* Indigenous
+* Women
+* Members of the LGBTQ+ community
+* Being from a working-class background
+* Those who struggle with mental illnesses
+* Having a disability whether visible or not
+* 
+Based on the Rooney rule, we are committed to not only consider a person of an underrepresented minority, but to bring at least one of them into the [final stage of the interview process](/handbook/people/hiring-process/#posthog-superday). In order to be successful with our approach, we focus on diversifying the top of the recruitment funnel. We are committed to not making an offer until we have brought an underrepresented candidate into the final stages. 
 
-We view increased inclusion into tech as a wonderful ethical benefit of our policies, but it's not the *fundamental reason* for encouraging diversity at PostHog. Underrepresented great talent is *particularly great talent*, that *any* business should want.
+We are currently trialing this approach and we still have some limitations to overcome:
+* We don’t currently track diversity data as part of the application process. While some characteristics of underrepresentation will be visible in the interview process, others are not. 
```

Comment:
Side thought: I'm nervous it feels meaningless when we don't ask candidates what they identify as (as then we're being presumptive based on the interview), but I think it's valid that you're stating this is a work in progress.

Reply:
I raised this with Eltje the other day - I believe she does check with candidates what they identify as.

Reply:
The second to gather this kind of data, you potentially open yourself up to discrimination law suits, even when we reject for other reasons. Tried this in the past and we had to stop based on legal advise. 

As Yakko mentioned, I started asking for pronouns for new joiners, but not as part of the interview process

Reply:
That makes sense - can see that for sure.